src/pages/_components/ExampleShowcase.tsx

```diff
@@ -168,6 +168,8 @@ export default function ExampleShowcase() {
 							</p>
 							<img
 								src={item.image.src}
+								width={item.image.width}
```

Comment:
Lets mark these as with `loading="lazy"` and `decoding="async"` as well since these are always below the fold on an initial page load

I could be sold on leaving the first image with eager loading since it's higher priority 🤷

Reply:
Done in 58fb0a0.

I think we’re cool to whack `lazy` on all of these given it’s lower down the page for now.